fast_api/routes/organization.py

```diff
@@ -342,3 +331,48 @@ def archive_admin_message(
 def set_language_display(request: Request, body: UserLanguageDisplay = Body(...)):
     user_manager.update_user_language_display(body.user_id, body.language_display)
     return pack_json_result({"data": {"changeUserLanguageDisplay": {"ok": True}}})
+
+
+@router.post("/mapped-sorted-paginated-users")
+def get_mapped_sorted_paginated_users(
+    request: Request, body: MappedSortedPaginatedUsers = Body(...)
+):
+    active_users = user_manager.get_active_users(body.filter_minutes, None)
```

Comment:
Every endpoint we add for the backend, which returns admin sensitive data must also be protected by check admin access as for earlier (simplified) active_users. Otherwise we create an information oracle for normal users with critical information exposed.